pkg/apis/mongodb/v1/mongodb_types.go

```diff
@@ -135,6 +138,12 @@ func (m MongoDBUser) GetUserName() string {
 	return m.Name
 }
 
+// Get the final SCRAM credentials secret-name by appending the user provided
+// scramsCredentialSecretName with "scram-credentials"
+func (m MongoDBUser) GetScramCredentialsSecretName() string {
```

Comment:
[nit] go doc strings should begin with the function name

from [the go docs](https://blog.golang.org/godoc)

`
Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.
`

I think we should rephrase in the form of  `GetScramCredentialsSecretName gets the final ...` etc